server/controllers/personalInfo.js

```diff
@@ -26,3 +26,15 @@ exports.getPersonalInfo = asyncHandler(async (req, res) => {
     res.status(500).json(err);
   }
 });
+
+exports.getInfoById = asyncHandler(async (req, res) => {
+  try {
+    const info = await PersonalInfo.findOne({
+      userId: req.params.id
+    }).populate('userId')
```

Comment:
While we are populating the user document, won't that return the user's password as well?

Reply:
Yes, it did return. I am changing to populate only profilePic in the next commit :)